dashboard/app/controllers/foorm/misc_survey_controller.rb

```diff
@@ -45,6 +45,31 @@ def new
       }
     end
 
+    def show
```

Comment:
can you add a comment to this method? So in the future we know the difference between show and new

Reply:
Semi-related nit -- maybe route could be `/form/:misc_form_path/show`?

Reply:
👍